Rubberduck.VBEditor.VB6/SafeComWrappers/VB/VBProjects.cs

```diff
@@ -92,23 +92,23 @@ void VB._dispVBProjectsEvents.ItemRemoved([MarshalAs(UnmanagedType.Interface), I
         void VB._dispVBProjectsEvents.ItemRenamed([MarshalAs(UnmanagedType.Interface), In] VB.VBProject VBProject,
             [MarshalAs(UnmanagedType.BStr), In] string OldName)
         {
-            var project = new VBProject(VBProject);
-
-            if (!IsInDesignMode())
+            using (var project = new VBProject(VBProject))
             {
-                project.Dispose();
-                return;
-            }
+                if (!IsInDesignMode())
+                {
+                    return;
+                }
 
-            var projectId = project.ProjectId;
+                var projectId = project.ProjectId;
 
-            var handler = ProjectRenamed;
-            if (handler == null || projectId == null)
-            {
-                project.Dispose();
-                return;
+                var handler = ProjectRenamed;
+                if (handler == null || projectId == null)
+                {
+                    return;
+                }
```

Comment:
this could be simplified to only checking the projectId and invoking the ProjectRenamed with an elvis-operator